src/components/Popups/PopupDrawingToolbar.scss

```diff
@@ -22,64 +57,20 @@
     display: flex;
     align-items: center;
     justify-content: center;
-
-    &::after {
-        position: absolute;
-        right: 0;
-        width: 1px;
-        height: 100%;
-        background-color: #000;
-        content: '';
-    }
+    border-right: 1px solid #000;
```

Comment:
I think this will leave an extra black border on the edge of the popup. We should use a sibling selector `& + &` and border-left, instead.

Reply:
The last item in the toolbar which is the add comment isn't in a group so that shouldn't happen. But I think you bring up a good point, I'll make the sibling selector change